assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/edit.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import { Disabled } from '@wordpress/components';
+import type { BlockEditProps } from '@wordpress/blocks';
+import { WC_BLOCKS_IMAGE_URL } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import { BlockSettings } from './block-settings';
+import type { BlockAttributes, Context } from './types';
+
+export const Edit = ( {
+	attributes,
+	setAttributes,
+	context,
+}: BlockEditProps< BlockAttributes > & Context ) => {
+	const blockProps = useBlockProps();
+
+	const Placeholder = () => {
+		return (
+			<>
+				{ context.thumbnailsPosition !== 'off' && (
```

Comment:
```suggestion
				{ context.thumbnailsPosition !== 'ThumbnailsPosition.Off' && (
```

Reply:
We can remove from here as well

```suggestion
				{ context.thumbnailsPosition !== ThumbnailsPosition.Off && (
```